packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1308,14 +1309,28 @@ class _TabBarState extends State<TabBar> {
 
     if (widget.isScrollable) {
       _scrollController ??= _TabBarScrollController(this);
-      tabBar = SingleChildScrollView(
-        dragStartBehavior: widget.dragStartBehavior,
-        scrollDirection: Axis.horizontal,
-        controller: _scrollController,
-        padding: widget.padding,
-        physics: widget.physics,
-        child: tabBar,
-      );
+      if (Platform.isAndroid && widget.physics == null) {
+        tabBar = ScrollConfiguration(
+          behavior: _NoGlowScrollBehaviour(),
```

Comment:
This does not take into account the ScrollBehavior that may currently be active. This _NoGlowScrollBehavior extends ScrollBehavior, but what if the actual scroll behavior of the app is CupertinoScrollBehavior, MaterialScrollBehavior, or a custom one created by the user? That would be ignored here since this is a different class.

ScrollBehavior actually provides a method that will let you modify the inherited behavior:

https://github.com/flutter/flutter/blob/4914a4090c0a3e3973200e966733583a60b5674e/packages/flutter/lib/src/widgets/scroll_configuration.dart#L59